_posts/2023-09-07-developing-functions-locally.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "The faster way to iterate on your OpenFaaS functions"
+description: "Learn how the local-run and watch features of faas-cli can help you iterate on functions faster."
+date: 2023-09-07
+categories:
+- faas-cli
+- openfaas
+- functions
+- devex
+dark_background: true
+author_staff_member: han
+# image: "/images/2023-07-k8s-jwt-iam/background.png"
```

Comment:
Missing canva? Did my link work for you yesterday?

Reply:
Yes the image has been added.